Test/comp/Variance.dfy

```diff
@@ -35,7 +35,8 @@ datatype Non<T> = Non(T){
     method mD(x: T) returns (r: T) { r := x; }
 }
 
-datatype Con<-T> = Con(T -> int) {
+// Con.java would create an error in Windows file systems because "Con" is a reserved file name.
```

Comment:
```suggestion
```
Thanks for the explanation, but I don't think we need to keep it around. This has the nice side-effect of preserving the line numbers in the `.expect`.

Reply:
Done. Indeed, if someone ever renamed it to `Con`, Dafny would generate a suitable error now on Windows, so this is not needed anymore.